.travis.yml

```diff
@@ -114,6 +114,9 @@ install:
       export PATH=$CYTHON_EXE:$PATH;
       brew link --overwrite python;
     fi
+  - if [ "$BUILD_DOCUMENTATION" = "ON" ] && [ "$CC_FOR_BUILD" = "gcc" ]; then
+      $PIP install --user sphinx sphinx_rtd_theme jupyter nbconvert --upgrade;
```

Comment:
Actually there are two reasons to have this here: 
* it refers to installation steps
* need to install this before `cmake ..` otherwise no sphinx target